### ISSUE TITLE
Auto Imports / Suggestions not working since adding typescript

### ISSUE THREAD

**Issue by developerprv** [microsoft/TypeScript#48210]

**Comment by mjbvz** [comment]
Based on the description, this issue sounds similar to #48165

**Comment by andrewbranch** [comment]
Yeah, this is a duplicate of #48165. It sounds like automatic type acquisition isn’t working—you should be able to work around it in the meantime by npm installing `@types/react`.

**Comment by developerprv** [comment]
> Yeah, this is a duplicate of #48165. It sounds like automatic type acquisition isn’t working—you should be able to work around it in the meantime by npm installing `@types/react`.

I tried this on the reacttest repo that I created. It did not work.

**Comment by andrewbranch** [comment]
@developerprv I’m about to put up a fix for #48165, but I can’t repro your issue at all. `node_modules/@types/react` already exists in `reacttest` so it seems to work fine. Can you try with the [nightly JavaScript](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next) extension?

<img width="838" alt="image" src="https://user-images.githubusercontent.com/3277153/159092308-ddb54415-069c-467a-84b1-d47b7765793d.png">

**Comment by willisk** [comment]
experiencing the same issue. I just copy pasted some typescript file into my workspace (which I normally don't use) and now even after deleting this (although I'm not sure this is the root cause), I can't get any correct imports any more
![image](https://user-images.githubusercontent.com/38631399/161144130-5adea922-e012-4da3-b532-a90640c6b37e.png)

**Comment by developerprv** [comment]
I recently set up a new computer, I first used "npx create-react-app my-app" to create a new react project. All import suggestions showing correctly for a few weeks. After a few weeks, I setup a project with "npx create-react-app my-app --template typescript" and now all of my standard react projects no longer receive the correct import suggestions, but the react with typescript projects do receive them correctly. So, it seems like the issue is adding typescript via create-react-app?

**Comment by 1898182** [comment]
I am having a similar issue with vite w/ Typescript. I cannot `CTRL + SPACE` to get the suggestion to popup. 

![not suggesting](https://user-images.githubusercontent.com/47544564/223019337-d20c1a40-0f48-4a9f-b58d-ebbfc2849d5a.png)

However if I hover with my mouse I can get to the "Quick Suggestions" and finally import from react. It is very annoying as you can figure.

![suggesting-import](https://user-images.githubusercontent.com/47544564/223019348-02ad63f7-4511-4e1b-a93a-30815ab98425.png)

Here is my package.json dependencies if it's relevant.

![packagejson](https://user-images.githubusercontent.com/47544564/223019583-4085c06f-548c-473f-8c1d-cdec1d37191b.png)

**Comment by RyanCavanaugh** [comment]
If you're encountering issues with auto-import, please log a new bug with repro steps. Thanks!
